### PR TITLE
TFS 2017 Update2 script changes

### PR DESCRIPTION
**Pull request by nishantiitp**

### CODE REVIEW

**Comment by nishantiitp** [comment]
Committing scripts for TFS 2017 Update 2

File: PauseIndexing.ps1 [hunk 1]
```diff
@@ -0,0 +1,82 @@
+﻿[CmdletBinding()]
+Param(
+    [Parameter(Mandatory=$True, Position=0, HelpMessage="The Server Instance against which the script is to run.")]
+    [string]$SQLServerInstance,
+
+    [Parameter(Mandatory=$True, Position=1, HelpMessage="Configuration DB")]
+    [string]$ConfigurationDatabaseName,
+    
+    [Parameter(Mandatory=$False, Position=2, HelpMessage="Pause Indexing for Code, WorkItem or All.")]
+    [string]$EntityType = "All"
```
Comment (nschonni) [inline PauseIndexing.ps1]: Switches might be easier to use `PauseIndexing.ps1 -ServerInstance foo -ConfigurationDabatabase -All`

File: TFS_2017Update2/Common.psm1 [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+    }
+}
+
+function ValidateCollectionName
+{
+    [CmdletBinding()]
+    [OutputType([string])]
+    Param
+    (
+        [string] $SQLServerInstance,
```
Comment (nschonni) [inline TFS_2017Update2/Common.psm1]: Manditory?

File: TFS_2017Update2/Common.psm1 [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+    return $CollectionID
+}
+
+function IsExtensionInstalled
+{
+    [CmdletBinding()]
+    [OutputType([boolean])]
+    Param
+    (
+        [string] $SQLServerInstance,
```
Comment (nschonni) [inline TFS_2017Update2/Common.psm1]: Manditory?

**Pull request merged**